phpcs.xml

```diff
@@ -2,11 +2,7 @@
 <ruleset name="custom">
     <description>Custom PHPCS Standard, based on Drupal standards.</description>
 
-    <!--
```

Comment:
@AlexSkrypnyk Fixed the lint issue but I have added back lint of `civictheme` directory here _but_ you must have had a reason for moving to civictheme?

Either way I think we need to lint civictheme every commit and everytime a developer runs `ahoy lint` - let me know your thinking.

Reply:
This was excluded because civictheme has own phpcs required to run in an isolated ci build.

Including it here will run the lint twice with potentially different rules.

you can just remove the linting from civictheme dir completely and use the one in the root

Reply:
It was initially done this way to have more loose standards for non theme code

Reply:
Thanks will reinclude and remove civictheme theme's lint standards